contrib/for-tests/network-tester/webserver.go

```diff
@@ -213,12 +215,13 @@ func contactOthers(state *State) {
 	for i := 0; i < 15; i++ {
 		endpoints, err := client.Endpoints(*namespace).Get(*service)
 		if err != nil {
-			state.Logf("Unable to read endpoints for %v/%v: %v; will try again.", *namespace, *service, err)
+			state.Logf("Unable to read the endpoints for %v/%v: %v; will try again.", *namespace, *service, err)
 			time.Sleep(time.Duration(1+rand.Intn(10)) * time.Second)
 		}
 
-		for _, e := range endpoints.Endpoints {
-			contactSingle("http://"+e.IP+":"+string(e.Port), state)
+		for i, e := range endpoints.Endpoints {
+			state.Logf("Attempting to contact IP %s at endpoint number %d value %v", e.IP, i, e.Port)
+			contactSingle("http://"+e.IP+":"+strconv.Itoa(e.Port), state)
```

Comment:
why not just use fmt.Sprintf here and avoid the strconv and string concatenation entirely?